girder_worker/utils.py

```diff
@@ -10,6 +10,10 @@
 import six
 
 
+def is_internal(task):
+    return task.startswith('celery')
```

Comment:
This seems a little fragile? Shouldn't we at least search for the '.' as well? Why not search in set of names, as in ```task in BUILTIN_CELERY_TASKS```?

Reply:
Can we just check `task.__module__` for `celery.builtins` or something?